share/spice/dota2/content.handlebars

```diff
@@ -0,0 +1,149 @@
+{{#if isItem}}
+    <div style="float: left; width: 100px;">
+        <img src="{{image}}" title="{{name}}"/>
+    </div>
+    <div style="float: right; width: 500px;">
+        <h1 class="zci__header">
+            {{name}}
+        </h1>
+        <hr />
+        <div class="twenty text--secondary">
+            Cost
+        </div>
+        <div class="eighty">
+            <img src="http://cdn.dota2.com/apps/dota2/images/tooltips/gold.png" title="Gold"/> {{cost}}
```

Comment:
Seeing as there's so many, you could build proxied URLs in the JS and pass them along here and do something like:

`<img src="{{assets.gold.img}}" title="{{assets.gold.title}}" />`